scripts/scil_visualize_scatterplot.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+"""
+Script to display scatter plot between two maps (ex. FA and MD).
+This script can be also used for the correspondence between two Myelin maps
+(ex. ihMTR and MTsat). For the myelin, the WM and GM probability maps are
+used to threshold the myelin maps. Therefore, the --myelin --in_wm and --in_gm
+options must be added.
+
+For general scatter plot:
+>>> scil_visualize_scatterplot.py FA.nii.gz MD.nii.gz out_filename_image.png
+
+For Myelin scatter plot:
+>>> scil_visualize_scatterplot.py ihMTR_map.nii.gz MTsat_map.nii.gz
+    out_filename_image.png --myelin --in_wm probability_wm_map.nii.gz
+    --in_gm probability_gm_map.nii.gz
+
+To display specific label for myelin scatter plot used:
+    --label 'WM Threshold' --label_myelin 'GM Threshold'
+
+"""
+
+import argparse
+import copy
+
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_x_map',
+                   help='Map in x axis, FA or ihMTR for example.')
+    p.add_argument('in_y_map',
+                   help='Map in y axis, MD or MTsat for example.')
+    p.add_argument('out_png',
+                   help='Output filename for the figure.')
+    p.add_argument('--in_mask',
+                   help='Binary mask map. Use this mask to extract x and '
+                        'y maps value from specific map or region: '
+                        'mask_wm or roi_mask')
+
+    myelo = p.add_argument_group(title='Myelin options')
+    myelo.add_argument('--myelin', action='store_true',
+                       help='Compute and display specific scatter plot for '
+                            'myelin maps.')
+    myelo.add_argument('--in_wm',
+                       help='Probability WM map.')
+    myelo.add_argument('--in_gm',
+                       help='Probability GM map.')
+
+    scat = p.add_argument_group(title='Scatter plot options')
+    scat.add_argument('--title',
+                      default='Scatter Plot',
+                      help='Use the provided info to title name. '
```

Comment:
Preferably, "[...] info _for the_ title name,". Same thing for the description of the next arguments (x_label, y_label, ..., etc.).